### ISSUE TITLE
Remove references to external claims database

### ISSUE THREAD

**Issue by Anbranin** [umts/incidents#280]
There's a lot of code that references the claims database that we have no plans to ever connect to again. This came to my attention because a programmer attempted to use [this method](https://github.com/umts/incidents/blob/master/app/models/incident.rb#L162) in the console, which will not work and will never work because it references models that inherit from [this one](https://github.com/umts/incidents/blob/master/app/models/claims_table.rb).
Basically we need to just delete that model and any model that inherits from it (hint: they start with the word `claims`), and any methods that use those models or rely on them and any controller methods that call them etc...
